css-animationworklet/Overview.bs

```diff
@@ -204,10 +230,70 @@ interface AnimationWorkletGlobalScope : WorkletGlobalScope {
     </pre>
 </div>
 
+Note: The statelessness allows animation worklet to perform optimization such as producing multiple
+animation frames in parallel and performing very cheap teardown and setup. Using StatelessAnimator
+is highly recommended to enable such optimizations.
+
+StatefulAnimator Interface {#stateful-animator-desc}
+---------------------------
+
+This interface represent a stateful animator. This animator can have local state and animation
+worklet guarantees that it maintains this state as long as the stateful animator fulfills the
+contract required by this interface.
```

Comment:
(Perhaps we should say, "... as described following." or some such since on first reading its not clear what the contract is.)